.github/bin/release-plan/date-format.mjs

```diff
@@ -0,0 +1,32 @@
+export function nowFormatted() {
+	const now = new Date();
+
+	switch (now.getMonth()) {
+		case 0:
+			return `January ${now.getUTCDate()}, ${now.getUTCFullYear()}`;
```

Comment:
There definitely is something in node that just gives the month, I didn't spend time looking.